### PR TITLE
Amendments to enable the manual docker image build

### PR DESCRIPTION
**Pull request by hwilliams1**
Amendments to enable the manual docker image build

### CODE REVIEW

File: start.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+# Start script for disqualified-officers-delta-consumer
+
+APP_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+if [[ -z "${MESOS_SLAVE_PID}" ]]; then
+    source ~/.chs_env/private_env
+    source ~/.chs_env/global_env
+    source ~/.chs_env/disqualified-officers-delta-consumer/env
+
+    PORT="${DISQUALIFIED_OFFICERS_DATA_API_PORT}"
```
Comment (VaughanJackson123) [inline start.sh:12]: `DISQUALIFIED_OFFICERS_DELTA_CONSUMER_PORT`?

**Comment by VaughanJackson123** [comment]
Do we need a `Tiltfile` too?

File: start.sh (hunk 1 above)
Comment (hwilliams1) [reply start.sh:12]: Good spot, I'll change that now

**Comment by hwilliams1** [comment]
> 

It has a Tiltfile to allow dev mode in tile

**Comment by VaughanJackson123** [comment]
> > 
> 
> It has a Tiltfile to allow dev mode in tilt

OK, is that enough? I noticed that apps like `officer-delta-processor` had both a`Tiltfile.dev` and a `Tiltfile`?

**Review by VaughanJackson123 — APPROVED** [review]

**Pull request merged**